src/main/java/org/synyx/urlaubsverwaltung/absence/web/AbsenceOverviewViewController.java

```diff
@@ -105,14 +105,36 @@ public String absenceOverview(
 
         final Person signedInUser = personService.getSignedInUser();
 
-        final List<Department> departmentsOfUser = departmentService.getAllowedDepartmentsOfPerson(signedInUser);
-        model.addAttribute("departments", departmentsOfUser);
-
-        final String fallbackDepartment = departmentsOfUser.isEmpty() ? "" : departmentsOfUser.get(0).getName();
-        final List<String> preparedSelectedDepartments = rawSelectedDepartments.stream().filter(StringUtils::hasText).collect(toList());
-        final List<String> selectedDepartmentNames = preparedSelectedDepartments.isEmpty() ?
-            List.of(fallbackDepartment) : preparedSelectedDepartments;
-        model.addAttribute("selectedDepartments", selectedDepartmentNames);
+        final List<Person> overviewPersons;
+        final List<Department> allDepartments = departmentService.getAllDepartments();
+        if (!allDepartments.isEmpty()) {
+
+            final List<Department> departments;
+            if (signedInUser.hasRole(BOSS) || signedInUser.hasRole(OFFICE)) {
+                departments = allDepartments;
+            } else {
+                departments = departmentService.getAllowedDepartmentsOfPerson(signedInUser);
+            }
+            model.addAttribute("departments", departments);
```

Comment:
```suggestion
            model.addAttribute("visibleDepartments", visibleDepartments);
```

Eventuell klarer als Abgrenzung zu den `selectedDepartments`